packages/openactive-integration-tests/test/helpers/request-helper.js

```diff
@@ -169,6 +169,7 @@ class RequestHelper {
 
   createHeaders() {
     return {
+      'Accept': 'application/vnd.openactive.booking+json; version=1',
```

Comment:
If I've understood content negotiation properly, this is needed for all requests to Bookings Systems